examples/data_sharing.tex

```diff
@@ -0,0 +1,210 @@
+\documentclass[a4paper]{article}
+\usepackage{data}
+
+%%%%% Now we input our variable terms as glossary entries %%%%%
+
+\newglossaryentry{type}{
+	name={Company},
+	first={the ``Company''},
+	text={Company},
+	description={is a commercial undertaking}
+}
+
+\newglossaryentry{undertaking}{
+	name={King of the Eagles},
+	first={King of the Eagles, hereafter `\glsentrytext{type}'},
+	text={the \glsentrytext{type}},
+	description={are Big birds. Live on a hill}
+}
+
+\newglossaryentry{processor}{
+	name={Mirkwood Elves},
+	first={Mirkwood Elves, hereafter `the Processor'},
+	text={Data Processor},
+	description={are the Elves of the Noldor Line}
+}
+
+\newcommand{\services}{Orc-slaying}
+
+\newcommand{\companyAddress}{123 Eirie Road}
+
+
+\newcommand{\processorAddress}{Third Tree, Mirkwood}
+
+\newcommand{\processorEmail}{elves@place.mw}
+
+\title{Data Sharing Agreement between \Glsentrytext{undertaking} \& \Glsentrytext{processor}}
+
+\begin{document}
+
+\begin{titlepage}
+
+\centering
+
+\vspace{0.4\textheight}
+
+\vspace{3cm}
+
+\scshape{\Huge\Glsfmtname{undertaking}}
+
+\vspace{0.1\textheight}
+
+\Large\scshape{and}
+
+\vspace{0.1\textheight}
+
+\scshape{\Huge\Glsfmtname{processor}}
+
+\vspace{0.2\textheight}
+
+
+\rule{0.5\textwidth}{0.4pt}
+
+Data Processing Agreement
+
+\rule{0.5\textwidth}{0.4pt}
```

Comment:
> @passamo9: There should be more space within the tramlines

Reply:
please use the tramline defined [here](https://github.com/passamo9/LaTeX_EnglandWales_Templates/blob/main/engwales-plead.sty#L29-L35), which we will extract to a shared util package later.

Reply:
It looks like you're suggesting a unified .sty file. That's a good idea.

The current styles don't match, so we have a number of options:

1. Manually copy those parts we want to match (I'm not sure what a 'tramline' is, but I suppose I could find it and make mine match other documents)
2. Have a unified style file, all of which are numbered, as required by the GDPR document.
3. Have some toggle saying 'this is a contract' or 'this is not a contract', which then changes the formatting on its own.
4. Have some separate .sty file, which contains only common elements, such as tramlines.

What do you think?

Reply:
>1. Manually copy those parts we want to match (I'm not sure what a 'tramline' is, but I suppose I could find it and make mine match other documents)

The tramline are the two lines above and below the document title:
<img width="570" alt="image" src="https://user-images.githubusercontent.com/69139574/108002452-f9939e00-6fef-11eb-8eb1-6bd4d1e426f5.png">

>2. Have a unified style file, all of which are numbered, as required by the GDPR document.

The GDPR document does not have standard numbering

Hope this helps!

Reply:
this was discussed in #16 and being implemented in #30.

> I'm not sure what a 'tramline' is

i had a link to the [code](https://github.com/passamo9/LaTeX_EnglandWales_Templates/blob/main/engwales-plead.sty#L29-L35).

Reply:
@yjyao The more efficient answer 😂